programs/libraries/README.md

```diff
@@ -0,0 +1,40 @@
+# Libraries
+Programs for adding helpful features not in native scarpet
+
+## Libraries in alphabetical order with creator:
+
+### compass_persist.sc:
+#### By Xendergo
```

Comment:
feels very specific to that compass app - not sure why this would need to be a generic use library.

Reply:
Idk either, ghoulboy asked for this, although I could've easily misunderstood: https://github.com/gnembon/scarpet/pull/177#discussion_r576392390

I'd be happy to remove it

Reply:
What were saying is forget about the library functions and event calls across apps. ALl the apps should be able to run in their own environment for ease of use, so just drop all of that.

Reply:
Ok, I've removed it then

Reply:
Ok, can you remove it from the ReadMe then?

Reply:
oh wait you did.